libs/remix-ui/editor/src/lib/remix-ui-editor.tsx

```diff
@@ -541,6 +542,31 @@ export const EditorUI = (props: EditorUIProps) => {
       }
     })
 
+    editor.onDidPaste((e) => {
+      if (e && e.range && e.range.startLineNumber >= 0 && e.range.endLineNumber >= 0 && e.range.endLineNumber - e.range.startLineNumber > 10) {
+        const modalContent: AlertModal = {
```

Comment:
Can this message be displayed once? So it doesn't get annoying to a user.

Reply:
done

Reply:
There is a spelling error in the warning message:
"...could loose all your money" should be
"...could LOSE all your money"